NGitLab.Mock/Repository.cs

```diff
@@ -538,6 +538,32 @@ public FileData GetFile(string filePath, string @ref)
         };
     }
 
+    public void GetRawFile(string filePath, Action<Stream> parser, GetRawFileRequest request)
+    {
+        var repo = GetGitRepository();
+        try
+        {
+            var @ref = request?.Ref ?? repo.Head.FriendlyName;
+            Commands.Checkout(repo, @ref);
+        }
+        catch (LibGit2Sharp.NotFoundException)
+        {
+            throw GitLabException.NotFound("File not found");
+        }
+
+        var fileCompletePath = Path.Combine(FullPath, filePath);
+
+        if (!System.IO.File.Exists(fileCompletePath))
```

Comment:
Please simplify:

```suggestion
        if (!File.Exists(fileCompletePath))
```

Reply:
Unfortunately, the fully-qualified name is required here as it conflicts with type `NGitLab.Mock.File`

Reply:
Fair enough.